package.json

```diff
@@ -46,16 +46,17 @@
   },
   "homepage": "https://github.com/concord-consortium/collaborative-learning#readme",
   "devDependencies": {
+    "@firebase/app-types": "^0.3.2",
     "@types/jest": "^23.3.1",
     "@types/jsonwebtoken": "^7.2.8",
     "@types/lodash": "^4.14.116",
     "@types/nock": "^9.3.0",
     "@types/query-string": "^6.1.0",
-    "@types/react": "^16.4.11",
+    "@types/react": "^16.4.13",
```

Comment:
Is `16.5.0` not available?

Reply:
Seems like not. Apparently it doesn't get released at the same time and/or they're not synchronizing version numbers.